test/angular/metadataReader.spec.ts

```diff
@@ -171,12 +171,14 @@ describe('metadataReader', () => {
     it('should work invoke Config.resolveUrl after all resolves', () => {
       let invoked = false;
       const bak = Config.resolveUrl;
-      Config.resolveUrl = (url: string) => {
-        invoked = true;
-        chai.expect(url.startsWith(normalize(join(__dirname, '../..')))).eq(true);
-        return url;
-      };
-      const code = `
+      try {
```

Comment:
I think we need to wrap with try/finally or the spy would leak when a test fails.